Classes/EventListener/ActsAfterInvokingEventListenerMethodsInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+namespace Neos\EventSourcing\EventListener;
+
+/*
+ * This file is part of the Neos.EventSourcing package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+use Neos\EventSourcing\Event\DomainEventInterface;
+use Neos\EventSourcing\EventStore\RawEvent;
+
+/**
+ * ActsAfterInvokingEventListenerMethodsInterface
+ *
+ * If this interface is implemented for an Event Listener class, the Event Publisher will call the respective
+ * method after invoking the actual event listener method (whenSomethingHappened()).
+ */
+interface ActsAfterInvokingEventListenerMethodsInterface extends EventListenerInterface
```

Comment:
Hmmm. I still dislike the `Acts...` name *if this extends EventListenerInterface* .Is there any reason why this "trait interface" also inherits from `EventListenerInterface`?

Maybe name suggestion: `AfterInvocationInterface`? :)